tilings/strategies/experimental_verification.py

```diff
@@ -0,0 +1,61 @@
+"""
+This file contains "experimental" verification packs, which are those that verify
+tilings for which we are not certain we can independently calculate their counting
+sequence or generating function.
+"""
+
+from typing import Iterable, Optional
+
+from comb_spec_searcher import VerificationStrategy
+from permuta import Perm
+from tilings import GriddedPerm, Tiling
+from tilings.algorithms import SubclassVerificationAlgorithm
+
+__all__ = [
+    "SubclassVerificationStrategy",
+]
+
+
+TileScopeVerificationStrategy = VerificationStrategy[Tiling, GriddedPerm]
+
+
+class SubclassVerificationStrategy(TileScopeVerificationStrategy):
+    """
+    Verify a tiling if its underlying ungridded permutations are contained in a subclass
+    of the search class.
+    """
+
+    def __init__(
+        self,
+        perms_to_check: Optional[Iterable[Perm]] = None,
+        ignore_parent: bool = True,
+    ):
+        if perms_to_check is None:
+            self.perms_to_check = None
+        else:
+            self.perms_to_check = set(perms_to_check)
+        super().__init__(ignore_parent=ignore_parent)
+
+    def verified(self, tiling: Tiling) -> bool:
+        assert self.perms_to_check is not None, "perms_to_check was never set"
+        algo = SubclassVerificationAlgorithm(tiling, self.perms_to_check)
+        return algo.verified()
+
+    @staticmethod
+    def formal_step() -> str:
+        return "tiling is a subclass"
```

Comment:
it might be useful to see which subclass. You can have the algo save perms left as a property perhaps?

Reply:
I don't exactly see how to do this. There's a single `SubclassVerificationStrategy` object doing all of the verification, right?

Reply:
This is a minor thing, so if it is too much effort we can skip. 

My understanding is that at the end of the verified method on `SubclassVerificationAlgorithm`, we know that it is a subclass of `Av(perms_left)`. 

So I see two options:
1) store the perms left as an attribute on the algorithm
2) have a method on the algorithm called `_perms_avoided` which returns `perms_left`. The verification is then simply `return bool(self._perms_avoided())`, and the string we want is f`is subclass of {Av(algo._perms_avoided())}'.